docs/source/development/sources.rst

```diff
@@ -1,68 +1,278 @@
 :orphan:
 
 Run From Sources
-=================
+========================================================================
 
 Environment Prerequisites
-~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Requirements:
 
 -  git
--  python3.6 for Ubuntu 18.04 and CentOS/RHEL 7
--  python3.8 for Ubuntu 20.04 and RockyLinux/CentOS/RHEL 8
+-  tig(optional interactive tui for git commits)
+-  python3.8 for Ubuntu 20.04 and RockyLinux/RHEL 8
+-  python3.9 for RockyLinux/RHEL 9
+-  python3.10 for Ubuntu 22.04
 -  pip, virtualenv, tox
--  MongoDB (http://docs.mongodb.org/manual/installation)
--  RabbitMQ (http://www.rabbitmq.com/download.html)
--  screen
+-  MongoDB
+   - Ubuntu (https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-ubuntu/)
+   - RHEL (https://www.mongodb.com/docs/v4.4/tutorial/install-mongodb-on-red-hat/)
+-  RabbitMQ / Erlang 26.x
+   - Ubuntu (https://www.rabbitmq.com/docs/install-debian)
+   - RHEL (https://www.rabbitmq.com/docs/install-rpm)
+-  Redis Stack (https://redis.io/docs/install/install-stack/linux/)
+-  tmux
 
 Ubuntu
-------
+------------------------------------------------------------------------
+
+Install required packages for python development.
+
+.. code-block:: bash
+
+    apt-get install python3-pip python3-venv gcc git make tmux libffi-dev libssl-dev python3-dev libldap2-dev libsasl2-dev
+
+
+Install a supported version of MongoDB.
 
 .. note::
-  For Ubuntu 20.04 replace with python3.8 equivalents
+  Mongo provides packages for Ubuntu 22.04 LTS ("Jammy") starting from MongoDB 6.0.4.  Adapt installation instructions according.
+
+.. code-block:: bash
 
+	apt-get install gnupg curl
+	curl -fsSL https://www.mongodb.org/static/pgp/server-4.4.asc | sudo gpg --dearmor -o /etc/apt/trusted.gpg.d/mongodb-server-4.4.gpg
+	sudo chmod 644 /etc/apt/trusted.gpg.d/mongodb-server-4.4.gpg
+
+	export VERSION_CODENAME=$(source /etc/os-release; echo $VERSION_CODENAME)
+	export DISTRO_ID=$(source /etc/os-release; echo $ID)
+	sudo cat <<EOF >/etc/apt/sources.list.d/mongodb-org-4.4.list
+	deb [ arch=amd64 ] https://repo.mongodb.org/apt/${DISTRO_ID} ${VERSION_CODENAME}/mongodb-org/4.4 multiverse
+	EOF
+	cat /etc/apt/sources.list.d/mongodb-org-4.4.list
+
+	sudo apt update
+	sudo apt-get install -y mongodb-org-server mongodb-org-shell mongodb-org-tools
+
+Install a supported version of RabbitMQ / Erlang
 
 .. code-block:: bash
 
-    apt-get install python-pip python-virtualenv gcc git make realpath screen libffi-dev libssl-dev python3.6-dev libldap2-dev libsasl2-dev
-    apt-get install mongodb mongodb-server
-    apt-get install rabbitmq-server
+	# configure repository
```

Comment:
This looks to be an odd code block that just says has 2 comments on it

Reply:
Good point!  I see no value in documenting how to install mongo/rabbitmq/redis on the dev machine, especially since the examples are Ubuntu centric.  It makes more sense to to link to the mongo/rabbitmq/redis installation sections of a production installation since it's the same steps.